### PR TITLE
feat(community-member): add component

### PR DESCRIPTION
**Pull request by kalinchernev**

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -48,6 +49,7 @@
         *   [✓ Slider](./framework/organisms/slider/README.md)
         *   [✓ Stats](./framework/organisms/stats/README.md)
         *   [✓ Twitter feed](./framework/organisms/twitter/README.md)
+        *   [✗ User profile](./framework/organisms/user-profile/README.md)
```
Comment (emeryro) [inline src/SUMMARY.md]: According to the screenshot, this is more a page than a component

**Comment by emeryro** [comment]
Don't we need a component for the right block in user profile? (contact me, languages and social media)

File: src/SUMMARY.md (hunk 1 above)
Comment (kalinchernev) [inline src/SUMMARY.md]: I would categorize it as a page if we had also the header and the footer and/or the sidebars.

**Comment by kalinchernev** [comment]
> Don't we need a component for the right block in user profile? (contact me, languages and social media)
I will separate that

**Comment by kalinchernev** [comment]
@emeryro the block with the social media is split into `contacts`.
I haven't split out the block with telephones, etc. as I think it could be a text with paragraphs or a table with the [highlight as background](https://ec-europa.github.io/farnet-styleguide/framework/atoms/highlight/#highlight-markup).

**Pull request merged**